libraries/chain/include/graphene/chain/asset_object.hpp

```diff
@@ -305,6 +306,12 @@ namespace graphene { namespace chain {
                 const_mem_fun<asset_object, bool, &asset_object::is_market_issued>,
                 member< object, object_id_type, &object::id >
             >
+         >,
+         ordered_unique< tag<by_issuer_and_id>,
```

Comment:
You can replace the `by_issuer` index with this one.

Reply:
correct, thank you https://github.com/bitshares/bitshares-core/pull/1749/commits/89949957341c91acfa16053d90870bfb9a49dbea